components/kyma-environment-broker/internal/hyperscaler/azure/tags.go

```diff
@@ -4,7 +4,7 @@ const (
 	// Azure tags used to identify the Kyma runtime
 	TagSubAccountID = "SubAccountID"
 	TagInstanceID   = "InstanceID"
-	TagOperationID  = "OperationID"
+	TagOperationID  = "OrchestrationID"
```

Comment:
Why is it changed? This value is used in `event_hub_step.go` to create Azure resources associated with a Runtime and it comes from processed provisioning/deprovisioning operation:
https://github.com/kyma-project/control-plane/blob/682586b01cfd0cfb270d995e6f696c9777f9c776/components/kyma-environment-broker/internal/process/provisioning/event_hub_step.go#L97-L102

Reply:
yeah, intellij refactor gone too far :D